source/documentation/adrs/adr-008.html.md.erb

```diff
@@ -0,0 +1,89 @@
+---
+title: ADR-008 Deployment environments
+last_reviewed_on: 2023-11-27
+review_in: 12 months
+---
+# <%= current_page.data.title %>
+
+### Date: 27-11-2023
+
+## Status
+
+🤔 Proposed
+
+## Context
+
+When working across multiple teams, or switching between teams, it can be hard to follow what each deployment environment is for and what practices it keeps. This can lead to data ending up in an improper environment, or tests being run with incorrect assumptions of the state of the environment.
+
+We sometimes need to test, demo or explore functionality between multiple connected services. To do so, we need persistent environments of each service that are connected together. Not all products have this, and those that do can be inconsistent, making it harder to perform these activities.
+
+## Decision
+
+All services must have the following environments:
+
+- Production
+- Preproduction
+- Demo
+- Local
+
+Services may commonly have additional environments:
+
+- Automatic PR environments
+- Manual PR environments
+- Other persistent environments
+
+### Production environment
+
+This is what our users access and it contains our real data. It must is the only environment in the production account.
+
+### Preproduction environment
+
+This has identical infrastructure to the production environment and is the only environment in the preproduction account. It is only accessible from within the MOJ Digital VPN. It is deployed in the pipeline before production to test integration that the release can be deployed successfully.
```

Comment:
Perhaps this could be "not accessible by the public" or "not accessible from the public internet". We already don't use the MoJ VPN, as it's not around anymore, and use global protect, but that's a solution not relevant to the decision.

Reply:
Cheers, I've gone for "not accessible by the public" since that's the underlying goal.